app/views/members/home/index.html.haml

```diff
@@ -1,7 +1,47 @@
 - content_for(:page, 'home')
 
 .row
-  .col-lg-6.col-12
+  .col-md-6
+    .card
+      .card-header
+        %i.fas.fa-fw.fa-stream
+        = I18n.t("admin.navigation.messages")
+
+    #posts
+      - @pinned.each do |post|
+        .card
+          .card-header{class: ('avatar-padding' if post.author.avatar.attached?)}
+            - if post.author.avatar.attached?
+              = image_tag post.author.avatar_representation, class: 'avatar'
+            %h4= post.author.name
+            %ul
+              %li= I18n.l(post.published_at, format: "%d %B %H:%m")
+              - post.tags&.each do |tag|
+                %li= tag
+          .card-body.content
+            = raw post.content
+
+      - @unpinned.each do |post|
+        .card
+          .card-header{class: ('avatar-padding' if post.author.avatar.attached?)}
+            - if post.author.avatar.attached?
+              = image_tag post.author.avatar_representation, class: 'avatar'
+            %h4= post.author.name
+            %ul
+              %li= I18n.l(post.published_at, format: "%d %B %H:%m")
+              - post.tags&.each do |tag|
+                %li= tag
+          .card-body.content
+            = raw post.content
+    
+    = render partial: 'layouts/partials/pagination', locals: {pagy: @pagination}
+
+  .col-lg-6.col-12.float-right
```

Comment:
this seems unnecessary

Reply:
The `.col-12` right? Otherwise its required for the info to be displayed on the right side

Reply:
Line 39 till line 44 it's double checkout balance now. looks like some old html from the old PR

Reply:
Ah yea, thanks